package.json

```diff
@@ -68,13 +58,8 @@
 	"pnpm": {
 		"peerDependencyRules": {
 			"ignoreMissing": [
-				"@babel/core",
-				"@emotion/react",
-				"eslint",
-				"prettier",
-				"react",
-				"react-dom",
-				"typescript"
+				"@types/node",
```

Comment:
How do we decide what to ignore here?

Reply:
these are things are inherently repo-wide ( think). e.g. node version can only be the version in the root nvmrc, so this `@types/node` will always be this version.